components/Theme.tsx

```diff
@@ -0,0 +1,19 @@
+import {useEffect, useState} from "react";
+
+const Theme = (theme?: string) => {
+  const [isReady, setIsReady] = useState(false);
+  useEffect( () => {
+    if (!theme && window.matchMedia("(prefers-color-scheme: dark)").matches) {
+      document.documentElement.classList.add("dark");
+    } else {
+      document.documentElement.classList.add(theme);
+    }
+    setIsReady(true);
+  }, []);
+
+  return {
+    isReady
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7734611)
```suggestion
    isReady,
```